pilz_testutils/include/pilz_testutils/joint_state_publisher_mock.h

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Why does this show up as a new file? As far as I understand this is just moved from prbt_hardware_support. Did you do `git mv`